client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ *
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import EmailVerificationCard from 'my-sites/domains/domain-management/components/email-verification';
+import { checkInboundTransferStatus, resendInboundTransferEmail } from 'lib/domains';
+
+class InboundTransferEmailVerificationCard extends React.Component {
+	static propTypes = {
+		selectedDomainName: PropTypes.string.isRequired,
+		selectedSiteSlug: PropTypes.string.isRequired,
+	};
+
+	state = {
+		email: '',
+		loading: true,
+	};
+
+	componentWillMount() {
+		this.refreshContactEmail();
+	}
+
+	componentWillUpdate( nextProps ) {
+		if ( nextProps.selectedDomainName !== this.props.selectedDomainName ) {
+			this.refreshContactEmail();
+		}
+	}
+
+	refreshContactEmail = () => {
+		this.setState( { loading: true } );
+
+		checkInboundTransferStatus( this.props.selectedDomainName, ( error, result ) => {
+			if ( ! isEmpty( error ) ) {
+				return;
+			}
+
+			this.setState( {
+				contactEmail: result.admin_email,
+				loading: false,
+			} );
+		} );
+	};
+
+	render() {
+		const { selectedDomainName, selectedSiteSlug, translate } = this.props;
+		const { loading, contactEmail } = this.state;
+
+		if ( loading ) {
+			return null;
+		}
+
+		return (
+			<EmailVerificationCard
+				contactEmail={ contactEmail }
+				verificationExplanation={ translate(
+					'We need to check your contact information to make sure you can be reached. Please verify your ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working. {{learnMoreLink}}Learn more.{{/learnMoreLink}}' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).